pages/manager/PluginHandler.inc.php

```diff
@@ -68,6 +68,21 @@ function plugins($args, &$request) {
 		$templateMgr->assign('helpTopicId', 'journal.managementPages.plugins');
 
 		$site =& $request->getSite();
+		$pluginDir = dirname(dirname(dirname(__FILE__))).'/plugins/';
+		// Don't show the install link unless something can be installed somewhere
```

Comment:
The "safer" component is a legitimate critique of https://github.com/ulsdevteam/pkp-lib/commit/68b8bf10c082d74b82bc12bd5fd0410d48f92d08#diff-77788b18f15e41b72c24abb00cba4e07R318 for Upgrade purposes, but I think the Install should be fine.  Unless an "Install" of a existing plugin is really an "Upgrade"?  I haven't actually checked.

Not using the web interface for plugin Installs/Upgrades/Deletes leaves me a bit less familiar; I was primarily responding to recent internal and external questions about the appearance of these links in contexts when they were known not be be relevant.